hyper-operations.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-08/hyper-operations-FAKE#",
+    "$id": "http://json-schema.org/draft-08/hyper-operations-FAKE",
+    "$recursiveRoot": true,
+
+    "title": "FAKE DEMO JSON Hypermedia Operations FOR RECURSION EXAMPLE",
+    "$ref": "http://json-schema.org/draft-08/hyper-schema",
```

Comment:
Did we resolve the issue of `$ref` having sibling keywords?

Reply:
Looks like there's an informal decision in #523.  I guess you're assuming that?

Reply:
Yes, there's also PR #585 which will hopefully start moving again soon.  I've been acting as if it's done because that's definitely what draft-08 will look like.  (I'm not sure what you mean by "informal decision", do we even have formal decisions here?  Other than merging PRs?)